MFVNeutralino/python/VertexSelector_cfi.py

```diff
@@ -118,6 +153,11 @@
 mfvSelectedVerticesTightNtk5    = mfvSelectedVerticesTight.clone() # for looping convenience
 
 mfvSelectedVerticesSeq = cms.Sequence(
+    mfvSelectedVerticesTightNoBeamPipe *
+    mfvSelectedVerticesTightNoDBS *
+    mfvSelectedVerticesTightNoNtk *
+    mfvSelectedVerticesTightNoDBSerr *
```

Comment:
I thought for all of these that we can just do something like https://github.com/DisplacedVertices/cmssw-usercode/blob/master/MFVNeutralino/test/histos.py#L28-L118 to get the N-1 versions. But maybe there's a use case I'm not thinking of, or maybe this is just easier and we should drop the one in histos.py...

Also, should `DBS` be `DBV` in this code?

Reply:
Thanks a lot for the information! I have removed this part now.